README.md

```diff
@@ -214,6 +220,27 @@ The [SolaceProtoBuffRecordTest](solace-apache-beam-samples/src/main/java/com/sol
     ```
 1. Verify that the messages are being outputted in the log.
 
+#### SolaceBigQuery
+
+The [SolaceBigQuery](solace-apache-beam-samples/src/main/java/com/solace/connector/beam/examples/SolaceBigQuery.java) example receives a message from PubSub+ queue, transforms it, and then writes it to a BigQuery table. 
+
+1. Create a BigQuery table with appropriate schema.
```

Comment:
Could you describe what exactly is the "appropriate schema"? This is to help people reading the document to easily run the sample without needing to dig around too much.